public/js/saveApplicant.js

```diff
@@ -44,22 +44,13 @@ let makeApiRequest = async (data) => {
 }
 
 let validateFormInputs = (data) => {
-    let validate = []
+    let validate = [];
 
-    validate.push(evalLength(data.name))
-    validate.push(evalLength(data.email))
-    validate.push(evalLength(data.phoneNumber))
-    validate.push(evalLength(data.whyDev))
-    validate.push(evalLength(data.codeExperience))
-    validate.push(evalLength(data.hearAboutId))
```

Comment:
Is there a reason why we don't have this anymore?

Reply:
We took it out as it wasn't doing anything. hearAboutId had been changed to a drop-down.